server/requirements.txt

```diff
@@ -5,3 +5,4 @@ git+git://github.com/joehand/flask-s3@8fe9f27ebb5251b7cf53a6684595d41dc4af2fbf#e
 # Superdesk
 -e git+git://github.com/liveblog/superdesk-core@a117d79e7fa10c92acf17aff957d05fdf2ac3f9b#egg=Superdesk-Core==0.0.1-dev
 -e git+git://github.com/liveblog/superdesk@33029d9fd55f7c7047a60abdac1f1911f044a27a#egg=Superdesk-Server
+norecaptcha-python3
```

Comment:
yes, it can be merged now


On Thu, Dec 17, 2015 at 12:10 AM, nistormihai <notifications@github.com>
wrote:

> In server/requirements.txt
> <https://github.com/superdesk/liveblog/pull/269#discussion_r47839009>:
>
> > @@ -5,3 +5,4 @@ git+git://github.com/joehand/flask-s3@8fe9f27ebb5251b7cf53a6684595d41dc4af2fbf#e
> >  # Superdesk
> >  -e git+git://github.com/liveblog/superdesk-core@a117d79e7fa10c92acf17aff957d05fdf2ac3f9b#egg=Superdesk-Core==0.0.1-dev
> >  -e git+git://github.com/liveblog/superdesk@33029d9fd55f7c7047a60abdac1f1911f044a27a#egg=Superdesk-Server
> > +norecaptcha-python3
>
> @ancafarcas <https://github.com/ancafarcas> is this pr ready to be merge
> into my branch?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/superdesk/liveblog/pull/269/files#r47839009>.
>